lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -98,6 +101,28 @@ export function initPlatform({
   const platformConfig: PlatformResult = {
     endpoint: defaults.endpoint,
   };
+  try {
+    let bitbucketServerVersion: string;
+    if (process.env.RENOVATE_X_PLATFORM_VERSION) {
```

Comment:
Why does this need to be added?

Reply:
if the user provides this value, we can save on api call

Reply:
@rarkins We've same for gitlab.